cmd/kubeadm/app/cmd/util/cmdutil.go

```diff
@@ -141,3 +141,12 @@ func GetClientSet(file string, dryRun bool) (clientset.Interface, error) {
 	}
 	return kubeconfigutil.ClientSetFromFile(file)
 }
+
+// GetFromConfigOrFlags checks if the "name" flag has been set, returns the value of flag on yes, otherwise returns the value from config.
```

Comment:
```suggestion
// ValueFromFlagsOrConfig checks if the "name" flag has been set. If yes, it returns the value of the flag, otherwise it returns the value from config.
```